### ISSUE TITLE
Mobile - Sync not working (Notebooks/Notes won't appear)

### ISSUE THREAD

**Issue by Grithubber** [laurent22/joplin#4900]
Hi there!

I've been using Joplin flawlessly since January, on both mobile (Android) and desktop, using Dropbox as the sync target.

Changed Linux distribution on my PC (from KDE Manjaro to KUbuntu). It's been a nightmare since (with Joplin).

What I have achieved so far: finally synced the desktop app. Can't sync the mobile app. It says it's syncing, but when it's done, it remains empty.

Please find the following debug log. Should I post the profile log on here too?

[syncReport-1619609863513.txt](https://github.com/laurent22/joplin/files/6391968/syncReport-1619609863513.txt)

Thanks!

**Comment by Grithubber** [comment]
Hi again!

Problem solved. This is what I did:

1. I exported all my desktop app notes as _RAW - Joplin Export Directory_;
2. I cleared the desktop app (deleted all notebooks/notes within it);
3. I re-imported the _RAW - Joplin Export Directory_;
4. It synced with the mobile app like a charm.

My problem was that I thought that sync would rework directly with the Joplin directory located in Dropbox. In other words, I used the notes located in Dropbox as backup (it does work if you copy the directory on your computer, and the target sync for the desktop app is a file system - this method did not work with the Android mobile app - I have Android 8.0).

Thank you! I've been working on this since Sunday, and I REALLY wanted to stay with Joplin and not go back to Evernote!

Can't wait for that centralized Joplin server :)

**Comment by Grithubber** [comment]
Hey again!

One thing that didn't work: tags aren't syncing to the mobile app.

Here's my latest debug report: [syncReport-1619638154379.txt](https://github.com/laurent22/joplin/files/6394353/syncReport-1619638154379.txt)

Thanks!

**Comment by roman-r-m** [comment]
Maybe you haven't waited long enough for the sync to finish? You need to have Joplin open to sync, there's no background sync atm.

**Comment by Grithubber** [comment]
Hi Roman!

Are you talking about the tags?

I always wait for sync to finish. I figured that out, but I didn't know it was because of the absence of background sync capability - thank you for that info!

**Comment by roman-r-m** [comment]
> Are you talking about the tags?

Yes, tags are synced the same way as everything else. Though it does seems strange if only tags are missing.

**Comment by Grithubber** [comment]
I left it a while, but the tags still aren't syncing.

At this point, it's ok with me. I don't need the tags on the mobile app. I usually use Simplenote on mobile for jotting down notes on the go. Then on the desktop, where I have both apps installed, I'll be copying and editing them to Joplin.

Still thought it would be worth mentioning. I'll let you know if they ever appear again.

**Comment by Grithubber** [comment]
Hi! I'm re-opening the issue just to let you know that I've been able to resolve it manually.

Basically, I deleted the previously existing tags on the desktop app. I recreated them on the mobile app, then assigned them back to the relevant notes. Both apps now sync wonderfully.

I am fortunate not to be using many tags in the first place, so doing it this way was easy for me.

Thank you and I hope this helps others stuck with the same problem.
